backend/app/data_migrations/populate_topic.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+from sqlalchemy.orm import Session
+
+from app.db.models import Response
+from .utils import has_rows, load_list
+
+
+# TODO: Rename Response -> Topic
+def populate_response(db: Session) -> None:
+    """Populate responses from CSV file."""
```

Comment:
The comment here references a csv where we are using json (same as some other functions)